README.md

```diff
@@ -3,94 +3,6 @@
 Recipes and code samples for
 [Google Cloud Service Extensions](https://cloud.google.com/).
```

Comment:
Can you:
- mention the difference between callouts and plugins
- add a link to the plugins README so it's discoverable
- update the docs link to point here: https://cloud.google.com/service-extensions/docs/overview

Reply:
For the plugins/callouts overview, you should probably copy the text from here: https://cloud.google.com/service-extensions/docs/overview#service-extensions-types

Reply:
Done!